apps/meteor/app/ui-master/server/inject.ts

```diff
@@ -76,27 +76,33 @@ export const injectIntoHead = (key: string, value: Injection): void => {
 };
 
 export const addScript = (key: string, content: string): void => {
+	const currentHash = crypto.createHash('sha1').update(content).digest('hex');
+	key += `_${currentHash}`;
```

Comment:
the `key` provided to `injectIntoHead ` cannot have the hash, it must stay the same, otherwise multiple assets will be created everytime the content changes..

Reply:
And they will be added to the HTML? Not having the hash in the key can create issues with caching wrong files when doing rolling upgrades